loom/common/core/allocator.h

```diff
@@ -22,6 +22,9 @@
 #define _CORE_ALLOCATOR_H_
 
 #include "loom/common/core/assert.h"
+#include "jemalloc/jemalloc.h"
+#include <stdint.h>
```

Comment:
Hmm, it doesn't seem like `platform.h` includes it, a lot of other files seem to include `stdint.h` directly though, so maybe we should attack this in a separate PR?